apps/ledger-live-mobile/src/components/TabBar/TransferDrawer.tsx

```diff
@@ -239,39 +239,41 @@ export default function TransferDrawer({ onClose }: Omit<ModalProps, "isRequesti
   }
 
   return (
-    <Flex flexDirection="column" alignItems="flex-start" p={7} pt={9} flex={1}>
-      <ScrollView
-        alwaysBounceVertical={false}
-        style={{ width: "100%" }}
-        testID="transfer-scroll-list"
-      >
-        {buttonsList.map((button, index) => (
-          <Box mb={index === buttonsList.length - 1 ? 0 : 8} key={button.title}>
-            <TransferButton {...button} testID={button.testID} />
-          </Box>
-        ))}
-      </ScrollView>
+    <>
+      <Flex flexDirection="column" alignItems="flex-start" p={7} pt={9} flex={1}>
+        <ScrollView
+          alwaysBounceVertical={false}
+          style={{ width: "100%" }}
+          testID="transfer-scroll-list"
+        >
+          {buttonsList.map((button, index) => (
+            <Box mb={index === buttonsList.length - 1 ? 0 : 8} key={button.title}>
+              <TransferButton {...button} testID={button.testID} />
+            </Box>
+          ))}
+        </ScrollView>
+      </Flex>
       {readOnlyModeEnabled && !hasOrderedNano && (
         <BuyDeviceBanner
+          image="buyDoubleFlex"
           topLeft={
-            <Text color="primary.c40" uppercase mb={3} fontSize="11px" fontWeight="semiBold">
+            <Text color="neutral.c00" mb={6} minHeight={36} fontSize="14px" fontWeight="semiBold">
               {t("buyDevice.bannerTitle2")}
             </Text>
           }
-          style={{ marginTop: 36, paddingTop: 13.5, paddingBottom: 13.5 }}
           buttonLabel={t("buyDevice.bannerButtonTitle2")}
           buttonSize="small"
           event="button_clicked"
           eventProperties={bannerEventProperties}
           screen={screen}
-          {...IMAGE_PROPS_SMALL_NANO}
+          {...IMAGE_PROPS_SMALL_NANO_BOX}
```

Comment:
Please always spread objects above other props unless we want to override some existing properties. 
When you spread an object first you make it clear to everyone that we will not override anything from the props declared above.
The only exception is when you explicitly want to override some props.